Contact footer version branch

tsohleDev opened the pull request:
Adding the contact me section and a footer with my social links

tsohleDev committed:
branch

Kashif-Saleem-Ghuman committed:
Add styling to languages html/css,javascript,ruby usin css float technique

Kashif-Saleem-Ghuman committed:
Make change to style using csss float property

tsohleDev committed:
Giving the form attributes

Kashif-Saleem-Ghuman committed:
Add styling for contact form in contact.css and works.css

tsohleDev committed:
Merge branch 'main' of https://github.com/tsohleDev/Portfolio into ContacAndFooter

harshita02 committed:
final contact and footer changes

tsohleDev committed:
Add form

tsohleDev committed:
add formspree to form

tsohleDev committed:
Merge branch 'contact-footer-version-branch' of https://github.com/tsohleDev/Portfolio into contact-footer-version-branch

tsohleDev committed:
Fixing linter errors for css

Olamarx commented on index.html:
```diff
@@ -218,13 +219,12 @@ <h3 class="h3">Skills</h3>
+                <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+                <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
+                <textarea type="text" required placeholder="Enter your message here..." name="message" id="" class="message"> </textarea>
+                <input type="submit" maxlength="500" value="Start Collaboration" class="buttons contact__button">
```
- [ ] This maxlength attribute should be used in the textarea, not the submit button as we are not writing anything in the submit.

Olamarx commented on index.html:
```diff
@@ -218,13 +219,12 @@ <h3 class="h3">Skills</h3>
-                <input type="button" value="Start Collaboration" class="buttons contact__button">
+            <form action="https://formspree.io/f/mrgdozvp" method="post" class="contact__form">
+                <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+                <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
```
[OPTIONAL] Nice one adding the pattern attribute but emails should not be case-sensitive. This means it should not check if I input it in lowercase or uppercase. So far I input a valid email then it should work which is the work of the type attribute email. I did use lowercase and it was submitted. I will encourage you to take a second look at it.
### Check the warning message, the user will not even understand the format you are expecting so if you are going to continue with this pattern, you have to add the expected format message, it maybe through the placeholder.
![image](https://user-images.githubusercontent.com/86200356/199456741-1dbc8cd8-0839-4606-80fb-b7aff0b03baf.png)

Olamarx reviewed (changes requested):
Hi @tsohleDev, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/RcZlXVml8oKtoGQeqe/giphy.gif"/>

## Highlights
- Form submits as expected ✔️ 
- Linter passed ✅ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tsohleDev committed:
Fixing the contact form

giving the message a length limitation of 500 charectors and giving the email input a permission to accept uppercase email adresses and describing the format in the placeholder area

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @tsohleDev 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

Form submits as expected ✔️
Linter passed ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
